package.json

```diff
@@ -17,7 +23,16 @@
     "url": "https://github.com/kagisearch/browser_extensions/issues"
   },
   "homepage": "https://github.com/kagisearch/browser_extensions#readme",
-  "dependencies": {
-    "adm-zip": "0.5.9"
+  "devDependencies": {
+    "adm-zip": "0.5.10",
+    "nodemon": "2.0.22",
+    "rome": "12.1.3"
+  },
+  "engines": {
+    "node": "18.x",
+    "npm": "9.x"
+  },
+  "overrides": {
+    "semver": "7.5.2"
```

Comment:
This was _necessary_ to avoid a security vulnerability with `nodemon`'s latest version.